### PR TITLE
prepare master for the next release

### PR DESCRIPTION
**Pull request by vieux**
17.04.x was cut

### CODE REVIEW

File: api/swagger.yaml [hunk 1]
```diff
@@ -44,18 +44,19 @@ info:
...
     In previous versions of Docker, it was possible to access the API without providing a version. This behaviour is now deprecated will be removed in a future version of Docker.
 
     The API uses an open schema model, which means server may add extra properties to responses. Likewise, the server will ignore any extra query parameters and request body properties. When you write clients, you need to ignore additional properties in responses to ensure they do not break when talking to newer Docker daemons.
 
-    This documentation is for version 1.28 of the API, which was introduced with Docker 17.04. Use this table to find documentation for previous versions of the API:
+    This documentation is for version 1.29 of the API, which was introduced with Docker 17.05. Use this table to find documentation for previous versions of the API:
 
     Docker version  | API version | Changes
     ----------------|-------------|---------
+    17.04.x | [1.28](https://docs.docker.com/engine/api/v1.28/) | [API changes](https://docs.docker.com/engine/api/version-history/#v1-28-api-changes)
```
Comment (thaJeztah) [inline api/swagger.yaml]: probably best to make this `17.04.0` instead of `17.04.x`

**Comment by thaJeztah** [comment]
build failed due to https://github.com/docker/docker/pull/31844. retriggering

File: api/swagger.yaml (hunk 1 above)
Comment (vieux) [inline api/swagger.yaml]: don't you have hope ? :) it should be `.x`, we shouldn't bump API between patch releases

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**